tests/generate_test.js

```diff
@@ -29,24 +29,18 @@ describe('generateCSSRuleset', () => {
         }, '.foo{background-color:red !important;}');
     });
 
-    it('prefixes vendor props with a dash', () => {
+    it('prefixes transition to webkit', () => {
```

Comment:
This test name should be left the same. It was (originally) testing to make sure that our `WebkitTransition` to `-webkit-transition` conversion works.

Why does this only produce `-webkit-transition` prefixes now, and no `-ms-transition` or `-moz-transition`?